### PR TITLE
Grant the workflow the permission to write packages

### PR DESCRIPTION
**Pull request by danjelalura**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ on:
 permissions:
   contents: write
   pull-requests: write
+  packages: write
```
Comment (willynganga) [inline .github/workflows/release.yml:7]: Why is this needed?

Comment (danjelalura) [reply .github/workflows/release.yml:7]: I was receiving a 403 when the tool was trying to deploy the artifacts to the GitHub package registry. After reproducing and testing this in a test repository, this option was the only one that fixed the issue.

Comment (willynganga) [reply .github/workflows/release.yml:7]: Interestingly, the deploy workflow does not need this and works fine. 🤔

**Review by willynganga — APPROVED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (Thilo-Acrolinx) [reply .github/workflows/release.yml:7]: That was also my question, how come the deploy workflow works without this permission? 🤔
